### PR TITLE
Cookies: Provide a mechanism for per user control over cookies

### PR DESCRIPTION
**Pull request by sakjur**
**What is this feature?**

This focuses primarily on enabling access to Google Analytics for users that are in areas where analytics is regulated and require user consent. This doesn't add that consent system, but it allows users that have an external consent tracking system to record consents on the user's profile preferences within Grafana.

**Why do we need this feature?**

It comes up from time to time that using Grafana with Google Analytics enabled in a regulated area might be problematic. This aims to provide parts of the framework we need to solve that.

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ Alpha features might be changed or removed without prior notice.
 | `authnService`                     | Use new auth service to perform authentication                                                                                                                               |
 | `sessionRemoteCache`               | Enable using remote cache for user sessions                                                                                                                                  |
 | `alertingBacktesting`              | Rule backtesting API for alerting                                                                                                                                            |
+| `individualCookiePreferences`      | Support overriding cookie preferences on an individual level                                                                                                                 |
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md:98]: what does 'on an individual level' mean?

**Comment by DanCech** [comment]
Looks good to me, nice branch name ;)

File: docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md (hunk 1 above)
Comment (sakjur) [reply docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md:98]: user, should I substitute individual for user?

**Review by mildwonkey — APPROVED** [review]
👍 I think this is a sufficient solution for the time being, which we can revisit as more weird cookie preferences arise.

**Pull request merged**